### DISCUSSION TITLE
How to sort multiclass model predictions?

### DISCUSSION THREAD

**Discussion by minhtran2801** [ThilinaRajapakse/simpletransformers#1016] [Q&A]
Hi all,

I have a multi-class model and I use 4 paragraphs in a document for predictions. How would I sort their similarity in order? Thank you.

**Comment by luizniero** [comment]
Hellow. You need to change the predict() method to use a softmax function (predict is using a argmax). I made this change and pull request at: https://github.com/ThilinaRajapakse/simpletransformers/pull/1360
